src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -224,13 +225,31 @@ public boolean equals(Object obj) {
         return this.versionedAddressBook.getPersonAssignmentList(person);
     }
 
+    /**
+     * Switches the active person to the selected person and displays his/her assignment list
+     * if he/she is present within the person list. Updates to no active person and hence no
+     * assignment list displayed if the person specified does not exist in the list of persons
+     * in {@code versionedAddressBook}.
+     *
+     * @param person the person whose assignment list will be displayed
+     */
     @Override
     public void updateFilteredAssignmentList(Person person) {
         this.versionedAddressBook.changeActivePerson(person);
-        this.versionedAddressBook.updateAssignmentList(person);
+        this.versionedAddressBook.updateAssignmentList();
+    }
```

Comment:
to be honest, i think updateAssignmentList could be a more proper name, since actually we haven't implement a function that can filter the assignments into groups or what we want eg. only completed